src/gsy_e/gsy_e_core/rq_job_handler.py

```diff
@@ -132,8 +133,8 @@ def launch_simulation_from_rq_job(scenario: bytes, settings: Optional[Dict],
                        slot_length_realtime=slot_length_realtime,
                        kwargs=kwargs)
 
-        log.error("Finishing simulation with job_id: %s and configuration id: %s",
-                  job_id, gsy_e.constants.CONFIGURATION_ID)
+        log.info("Finishing simulation with job_id: %s and configuration id: %s",
```

Comment:
The same as [here](https://github.com/gridsingularity/gsy-e/pull/1461/files#r905921874) applies here too